specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Ah.  OK, so the import process is basically going to be importing both the election definitions and the CVR data at the same time.